Web Development/readme.md

```diff
@@ -90,6 +96,19 @@ This category is a curated collection of valuable web development resources cont
 </tr>
 </table>
 
+#### Deployment Roadmap
+
+<table width="100%">
+    <tr>
+    <th>Resource Name</th>
+    <th>Description</th>
+  </tr>
+<tr>
+   <td><a href="https://roadmap.sh/full-stack">Deployment</a></td>
+   <td>This is the roadmap for full-stack development with deployment</td>
+</tr>
+</table>
+
```

Comment:
I think you need to remove this, because deployment is already added in the full stack, and this resource is already exist in the [full stack](https://github.com/hars-21/ResourceHub/blob/main/Web%20Development/readme.md#full-stack-roadmap) roadmap section

Reply:
I have made the changes. Please review